dm/dm.go

```diff
@@ -72,13 +73,15 @@ var usage = func() {
 	fmt.Fprintln(os.Stderr, "Flags:")
 	flag.PrintDefaults()
 	fmt.Fprintln(os.Stderr)
-	os.Exit(1)
+	fmt.Fprintln(os.Stderr, "--stdin requires a file name and either the file contents or a tar archive containing the named file.")
+	fmt.Fprintln(os.Stderr, "        a tar archive may include any additional files referenced directly or indirectly by the named file.")
+	panic("\n")
 }
 
 func getGitRegistry() *registry.GithubRegistry {
 	s := strings.Split(*template_registry, "/")
 	if len(s) < 2 {
-		log.Fatalf("invalid template registry: %s", *template_registry)
+		panic(fmt.Errorf("invalid template registry: %s", *template_registry))
```

Comment:
I'm still not convinced that panic is the right thing here because the only we do is log the Fataln in case of failure and don't do any clean up work. So it still seems to me like we're just complicating the flow of things for no discernible benefit.
What am I missing? :)